modules/catalog/src/main/java/org/apache/ignite/internal/catalog/storage/NewZoneEntry.java

```diff
@@ -72,13 +72,17 @@ public CatalogEventParameters createEventParameters(long causalityToken, int cat
     public Catalog applyUpdate(Catalog catalog, long causalityToken) {
         descriptor.updateToken(causalityToken);
 
+        int defaultZoneId = catalog.defaultZone() != null
+                ? catalog.defaultZone().id()
+                : descriptor.id();
```

Comment:
I do not understand why we take `descriptor.id()` of a new zone as a defaultZoneId

Reply:
if default zone is not specified, the next created zone will be promoted as default. This basically means that the first created zone will become default one. Also, this will work as self-healing in case we lose default zone for some reason (at the moment due to a some failure)

Reply:
I don't think that this is a good idea to introduce such non-trivial behaviour. At least we must document such behaviour, so it doesn't come as a surprise for an user

Reply:
Discussed this in private and decided such behavior may obscure the problem, and, as result, the user may discover it too late when tables created in wrong zone have already a lot of data.

removed updating of default in NewZoneEntry